tests/native_tests/core.hy

```diff
@@ -41,7 +41,7 @@
          (except [TypeError] True)
          (else (assert False)))))
 
-(defn test-coll? []
+(defn test-collq []
```

Comment:
*sigh* We *really* need to work on our mangling...

FWIW pytest lets you [override the test prefix](https://docs.pytest.org/en/latest/customize.html#confval-python_functions). Maybe it'd be worth it to do that with something like:

```ini
[pytest]
# *insert comment lamenting state of world, etc.*
python_function = test_* is_test_*
```

Although it's a hack:

- It's not much more hacky than a questionable (haha!) `q` suffix on every single test function.
- It's a bit less weird to read. (random person looking through code: *Why are there random `q`s everywhere?*)
- The test names will make more sense. This is testing `coll?`, not `collq`...